format.md

```diff
@@ -23,33 +24,33 @@ The terminology for array types in this document is based on the
 
 ### Motivation
 
-Standard ways to represent or serialize vector geometries include WKT (e.g.
+Standard ways to represent or serialize vector geometries include WKT (e.g.,
 "POINT (0 0)"), WKB and GeoJSON. Each of those representations have a
 considerable (de)serialization cost, neither do they have a compute-friendly
 memory layout.
 
 The goal of this specification is to store geometries in an Arrow-compatible
-format that has 1) low (de)serialization overhead and 2) once in memory is
-cheap to convert to geospatial libraries (e.g. GEOS or JTS) or easy to
-directly operate on (e.g. directly working with the coordinate values).
+format that has (1) low (de)serialization overhead and (2) once in memory is
+cheap to convert to geospatial libraries (e.g., GEOS or JTS) or easy to
+directly operate on (e.g., directly working with the coordinate values).
```

Comment:
For readability I'd consider bringing this out into a numbered list, and I'd also suggest bringing `has` after `(1)` so that `that ... ` reads appropriately in both clauses.

```suggestion
format that 

1. has low (de)serialization overhead
2. once in memory is cheap to convert to geospatial libraries (e.g., GEOS or JTS) or easy to directly operate on (e.g., directly working with the coordinate values).
```